drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-marshaller/pom.xml

```diff
@@ -0,0 +1,214 @@
+                <arg>-jsmn=SCESIMMainJs</arg>
+              </args>
+              <plugins>
+                <plugin>
+                  <groupId>org.kogito</groupId>
+                  <artifactId>gwt-jsonix-schema-compiler</artifactId>
+                </plugin>
+              </plugins>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-javascript</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.hisrc.jsonix</groupId>
+                  <artifactId>jsonix-scripts</artifactId>
+                  <version>${jsonix-scripts.version}</version>
+                  <type>js</type>
+                  <classifier>all</classifier>
+                  <outputDirectory>${js.destination}</outputDirectory>
+                  <destFileName>Jsonix-all.js</destFileName>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>move-generated-sources</id>
+            <phase>process-sources</phase>
+            <configuration>
+              <target>
+                <copy file="${project.build.directory}/generated-sources/xjc/SCESIM.js" tofile="${js.destination}/SCESIM.js"/>
+              </target>
+            </configuration>
+            <goals>
+              <goal>run</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <!-- GWT Maven Plugin-->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>gwt-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>compile</goal>
+              <goal>generateAsync</goal>
+              <goal>test</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <module>org.drools.workbench.scenariosimulation.kogito.marshaller.ScesimMarshaller</module>
+          <style>PRETTY</style>
+          <compileReport>true</compileReport>
+          <soycDetailed>true</soycDetailed>
+          <logLevel>TRACE</logLevel>
+          <noServer>false</noServer>
+        </configuration>
+      </plugin>
+      <!-- DISABLE CHECKSTYLE -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-checkstyle-plugin</artifactId>
+        <configuration>
+          <skip>true</skip>
```

Comment:
shouldn't be disabled just for special folders?

Reply:
@jomarko good point, done!